RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_tools.py

```diff
@@ -225,6 +260,23 @@ def psetHadronicOverEMCut(wpEB, wpEE):
         isIgnored = cms.bool(False)
         )
 
+# Configure energy and pileup dependend H/E cut
+def psetHadronicOverEMEnergyScaledCut(wpEB, wpEE):
+    return cms.PSet( 
+        cutName = cms.string('GsfEleHadronicOverEMEnergyScaledCut'),
+        barrelC0 = cms.double( wpEB.hOverECut_C0 ),
+        barrelCE = cms.double( wpEB.hOverECut_CE ),
+        barrelCr = cms.double( wpEB.hOverECut_Cr ),
+        endcapC0 = cms.double( wpEE.hOverECut_C0 ),
+        endcapCE = cms.double( wpEE.hOverECut_CE ),
+        endcapCr = cms.double( wpEE.hOverECut_Cr ),
+        rho = cms.InputTag("fixedGridRhoFastjetAll"),
```

Comment:
is there a plan to use centralNeutral rho at some point?

Reply:
Not yet discussed. We`ll do in the next iteration of ID.